packages/codegen-ui-react/lib/primitive.ts

```diff
@@ -128,3 +129,29 @@ export const PrimitivesWithChangeEvent: Set<Primitive> = new Set([
   Primitive.SwitchField,
   Primitive.TextField,
 ]);
+
+export const PrimitiveDefaultPropertyValue: Partial<
+  Record<Primitive, { [property: string]: FixedStudioComponentProperty }>
```

Comment:
Not sure if it's more succinct or readable, but I think this can be defined as a PrimitiveLevelPropConfig<FixedStudioComponentProperty>

Reply:
Oh nice! I didn't notice we had that.